api/db/migrate/20250917114649_create_events.rb

```diff
@@ -0,0 +1,20 @@
+class CreateEvents < ActiveRecord::Migration[8.0]
+  def change
+    create_table :events do |t|
+      t.string  :title, null: false
+      t.text    :description
+      t.string  :location, null: false
+      t.datetime :start_date, null: false
+      t.datetime :finish_date
+      t.integer :participant_capacity
+      t.decimal :ticket_price, precision: 10, scale: 2
```

Comment:
Ticket price should not allow negative values at the database level. Add a database constraint for ticket_price >= 0.
```
t.check_constraint "ticket_price >= 0", name: "ticket_price_non_negative"
```